### PR TITLE
Remove label typing pollution from RawInput in FileInput

### PR DESCRIPTION
**Pull request by samshara**
### This PR ensures that there are
- [x] No unnecessary console logs
- [x] No unwanted comments
- [x] No typos
- [x] `yarn typecheck` shows no issue
- [x] `yarn lint` shows no issue

### This PR adds one or more new components / utils

- [x] Add the newly created component and its props to exports
- [x] Write story for newly created component(s)
- [x] No empty files

### This PR fixes or updates existing components / utils

- [x] Update / add story to reflect the changes
- [x] Doesn't introduce breaking changes

### Beta release vXX.XX.XX-beta.YY
- [x] Add a commit for beta release

### CODE REVIEW

**Review by frozenhelium — APPROVED** [review]

**Review by tnagorra — CHANGES_REQUESTED** [review]
There's conflict in package.json

**Review by tnagorra — APPROVED** [review]

File: src/components/FileInput/index.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const isValidFile = (fileName: string, mimeType: string, acceptString?: s
     });
 };
 
-type InheritedProps<T> = (Omit<InputContainerProps, 'input'> & Omit<RawInputProps<T>, 'onChange' | 'value'>);
+type InheritedProps<T> = (Omit<InputContainerProps, 'input'> & Omit<RawInputProps<T>, 'onChange' | 'value' | 'label'>);
```
Comment (tnagorra) [inline src/components/FileInput/index.tsx:30]: Why not do this change for other inputs as well?

**Review by tnagorra — CHANGES_REQUESTED** [review]
Let's do this change for all of the inputs or find a way so that this kind of problem doesn't occur next time in the future.

**Comment by tnagorra** [comment]
The issue this PR addresses has been fixed in another PR

**Pull request closed without merging**